x/tokenfactory/simulation/sim_msgs.go

```diff
@@ -15,8 +15,13 @@ import (
 
 // RandomMsgCreateDenom creates a random tokenfactory denom that is no greater than 44 alphanumeric characters
 func RandomMsgCreateDenom(k keeper.Keeper, sim *simtypes.SimCtx, ctx sdk.Context) (*types.MsgCreateDenom, error) {
+	acc, senderExists := sim.RandomSimAccountWithConstraint(accountCanPayTokenCreationFee(k, sim, ctx))
```

Comment:
Though, was it never passing before?

Its 100% fine to not always create valid messages

Reply:
Okay I will draft this then and add it to this PR. Sorry I slipped this in, didn't want to keep opening new PRs for small additions

Reply:
@ValarDragon It was passing before, but that was because we never ran into an address that used up all its stake tokens. Once that happens, without this we generate a message that cant be executed succesfully